dataland-frontend/src/components/forms/CreateSfdrDataset.vue

```diff
@@ -0,0 +1,308 @@
+import InputTextFormField from "@/components/forms/parts/fields/InputTextFormField.vue";
+import FreeTextFormField from "@/components/forms/parts/fields/FreeTextFormField.vue";
+import NumberFormField from "@/components/forms/parts/fields/NumberFormField.vue";
+import DateFormField from "@/components/forms/parts/fields/DateFormField.vue";
+import SingleSelectFormField from "@/components/forms/parts/fields/SingleSelectFormField.vue";
+import MultiSelectFormField from "@/components/forms/parts/fields/MultiSelectFormField.vue";
+import AddressFormField from "@/components/forms/parts/fields/AddressFormField.vue";
+import RadioButtonsFormField from "@/components/forms/parts/fields/RadioButtonsFormField.vue";
+import SubmitButton from "@/components/forms/parts/SubmitButton.vue";
+import SubmitSideBar from "@/components/forms/parts/SubmitSideBar.vue";
+import YesNoNaFormField from "@/components/forms/parts/fields/YesNoNaFormField.vue";
+import UploadReports from "@/components/forms/parts/UploadReports.vue";
+import DataPointFormField from "@/components/forms/parts/kpiSelection/DataPointFormField.vue";
+import PercentageFormField from "@/components/forms/parts/fields/PercentageFormField.vue";
+import ProductionSitesFormField from "@/components/forms/parts/fields/ProductionSitesFormField.vue";
+import { objectDropNull } from "@/utils/UpdateObjectUtils";
+import { smoothScroll } from "@/utils/SmoothScroll";
+import { DocumentToUpload, uploadFiles } from "@/utils/FileUploadUtils";
+import MostImportantProductsFormField from "@/components/forms/parts/fields/MostImportantProductsFormField.vue";
+import { Subcategory } from "@/utils/GenericFrameworkTypes";
+import ProcurementCategoriesFormField from "@/components/forms/parts/fields/ProcurementCategoriesFormField.vue";
+
+export default defineComponent({
+  setup() {
+    return {
+      getKeycloakPromise: inject<() => Promise<Keycloak>>("getKeycloakPromise"),
+    };
+  },
+  name: "CreateSfdrDataset",
+  components: {
+    SubmitButton,
+    SubmitSideBar,
+    UploadFormHeader,
+    SuccessMessage,
+    FailMessage,
+    FormKit,
+    Card,
+    PrimeButton,
+    Calendar,
+    YesNoFormField,
+    InputTextFormField,
+    FreeTextFormField,
+    NumberFormField,
+    DataPointFormField,
+    DateFormField,
+    SingleSelectFormField,
+    MultiSelectFormField,
+    NaceCodeFormField,
+    AddressFormField,
+    RadioButtonsFormField,
+    YesNoNaFormField,
+    PercentageFormField,
+    ProductionSitesFormField,
+    MostImportantProductsFormField,
+    ProcurementCategoriesFormField,
+    UploadReports,
+  },
+  directives: {
+    tooltip: Tooltip,
+  },
+  emits: ["datasetCreated"],
+  data() {
+    return {
+      formId: "createSFDRForm",
+      waitingForData: true,
+      dataDate: undefined as Date | undefined,
+      companyAssociatedSfdrData: {} as CompanyAssociatedDataSfdrData,
+      sfdrDataModel: sfdrDataModel,
```

Comment:
Useless assignment.  Can just be `sfdrDataModel`

Reply:
Removed